chalicelib/action.py

```diff
@@ -0,0 +1,50 @@
+# from flask import jsonify, Blueprint
```

Comment:
I'd rather not commit large swaths of commented out code. Please delete this file if possible.